### PR TITLE
include reasons in validation output message

### PR DESCRIPTION
**Pull request by rickardenglund**

### CODE REVIEW

File: problems/validation.go [hunk 1]
```diff
@@ -27,6 +31,17 @@ func (problem ValidationProblem) TrimEmpty() Problem {
 	return problem
 }
 
+func (problem ValidationProblem) Error() string {
+	sb := strings.Builder{}
+	sb.WriteString(fmt.Sprintf("%s ", problem.Title))
```
Comment (Ekberg) [inline problems/validation.go:36]:
```suggestion
	fmt.Fprintf(%sb, "%s ", problem.Title))
```

File: problems/validation.go [hunk 2]
```diff
@@ -27,6 +31,17 @@ func (problem ValidationProblem) TrimEmpty() Problem {
...
+
+	for i, reason := range problem.Reasons {
+		sb.WriteString(fmt.Sprintf("reason(%d): %s; ", i, reason.Error()))
```
Comment (Ekberg) [inline problems/validation.go:39]: I think we want to print the reason name and "reason" here. The intent of a ValidationReason is to be read with `name + " " + reason` The number is not relevant here (I think)

File: problems/validation.go (hunk 1 above)
Comment (rickardenglund) [reply problems/validation.go]: ah, nice :)

File: problems/validation.go (hunk 2 above)
Comment (rickardenglund) [reply problems/validation.go]: ah, ok. I added the number mostly in order to get some kind of delimiter. This is better.

File: problems/validation.go [hunk 3]
```diff
@@ -27,6 +31,17 @@ func (problem ValidationProblem) TrimEmpty() Problem {
 	return problem
 }
 
+func (problem ValidationProblem) Error() string {
+	sb := strings.Builder{}
+	fmt.Fprintf(&sb, "%s ", problem.Title)
+
+	for _, reason := range problem.Reasons {
+		fmt.Fprintf(&sb, "%s: ", reason.Error())
```
Comment (Ekberg) [inline problems/validation.go:39]:
```suggestion
		fmt.Fprintf(&sb, "%s, ", reason.Error())
```

**Review by Ekberg — APPROVED** [review]

**Pull request merged**